config/database.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'default' => env('DB_CONNECTION', 'processmaker'),
+    'default' => env('DB_CONNECTION', 'mysql'),
```

Comment:
Consider that some instances could have configure `DB_CONNECTION` in its `.env` file.
In order to keep compatibility consider to keep the name of the default connection as `processmaker`

If not possible:
Please add a note to the PR and Jira to remove the `DB_CONNECTION` from .env or change it to `mysql`